src/api/spec/mailers/event_mailer_spec.rb

```diff
@@ -155,8 +155,18 @@
         expect(mail.from).to eq([from.email])
       end
 
-      it 'renders the body' do
-        expect(mail.body.encoded).to have_text('Last lines of build log:')
+      context 'and there is a payload' do
+        it 'renders the body' do
+          expect(mail.body.encoded).to have_text('Last lines of build log:')
+        end
+      end
+
+      context 'but there is no payload' do
+        let(:faillog) { nil }
+
+        it 'renders the body' do
```

Comment:
This is true? I think it should be something like:

```suggestion
        it 'renders the body, but does not have a build log' do
```